server.js

```diff
@@ -0,0 +1,19 @@
+const axios = require("axios")
+const app = require("express")()
+const PORT = 3000
+
+// enable CORS
+app.use((req, res, next) => {
+  res.header("Access-Control-Allow-Origin", "*")
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept")
+  next()
+})
+
+app.get("/cat-facts", (req, res) => {
+  const url = "https://catfacts-api.appspot.com/api/facts?number=25"
+  axios.get(url).then(response => res.send(response.data))
+})
```

Comment:
Pretty sure express has a builtin cors middleware. The only other thing would to have your server and client be different subdomains of the same domain. Also i would take points off right now on security for allowing all requests from all origins